doc/faq.rst

```diff
@@ -101,6 +101,25 @@ We also retain logs of IP addresses for 30 days, which is used solely in
 the case of detecting abuse of the service. If you have suggestions for
 how we can ensure the privacy of our data and users, we'd love to hear it!
 
+How secure is mybinder.org?
+---------------------------
+
+The Binder team has put in a lot of work to ensure that the mybinder.org
+service runs as secure as possible. However, it is a free, public service
+that is open to the world, and **you should never share sensitive or personal
+information within a Binder repository**. This includes passwords, data that
```

Comment:
Maybe expand/clarify that this means don't store it somewhere that ends up in the build binder image (Dockerfile, git repo, gist, some link you download) but also that you should not enter this kind of information when you are using a binder (don't copy and paste your key or API token into a notebook)